### ISSUE TITLE
yaw/pitch/roll okay... but slave view needs to be rolled - how to do?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [LiquidGalaxy/liquid-galaxy#6]
```
Hi guys - just an update on that issue (#4 now closed) I was having with the 
slave view being rolled.

Actually the slave vector IS pointing in the right direction after the 
yaw-pitch-roll's are applied relative to the view master... so what I thought 
was a bug, isn't really. 

However, to be useful after the slave view is pointing in the right direction 
an addition "roll" operation is needed to reorient the view.

So, basically yaw/pitch/roll let's you point the slave in any direction 
relative to the master, but you need to be able to re-orient final view to 
finish things off. Perhaps there is an additional config for the slaves that 
can be used in the drivers.ini to do this?

Cheers, Andrew
```

Original issue reported on code.google.com by `alfski` on 8 Oct 2010 at 1:02
